scripts/utils/nexus_interaction.py

```diff
@@ -135,6 +135,7 @@ def update_composite_view(composite_view: Dict, rev: int, nexus_url: str, organi
     try:
         req = requests.put(
             nexus_url + '/views/' + organisation + '/' + project + '/' + parse.quote_plus(
+                # TODO: adapt so it also works for global search (id has a different base path)
```

Comment:
@kaij I figured out that this method of updating composite views also works for [global search](https://bluebrainnexus.io/docs/delta/api/search-api.html). However, the id has a different base path and the code would have to be adapted for use with global search.

Reply:
see #28